.external

```diff
@@ -1 +1 @@
-xamarin/monodroid:master@c494b1762213231794867e62deabb577ce121462
+dellis1972/monodroid:fixtestsmore@e9883e15000ad090ef4fb6c81eddf771b421fe29
```

Comment:
It should now be possible to instead bump to xamarin/monodroid/master@bd742d597b84083013e8a91e9c364dcb03e37131.